beetmoverscript/src/beetmoverscript/task.py

```diff
@@ -64,34 +67,34 @@ def _check_scopes_exist_and_all_have_the_same_prefix(scopes, prefixes):
         raise ScriptWorkerTaskException("Scopes must exist and all have the same prefix. " "Given scopes: {}. Allowed prefixes: {}".format(scopes, prefixes))
 
 
-def get_task_bucket(task, script_config):
-    """Extract task bucket from scopes"""
-    prefixes = _get_bucket_prefixes(script_config)
-    scopes = _extract_scopes_from_unique_prefix(task["scopes"], prefixes=prefixes)
-    buckets = [s.split(":")[-1] for s in scopes for p in prefixes if s.startswith(p)]
-    log.info("Buckets: %s", buckets)
+def get_task_resource(context):
+    """Extract task cloud resource from scopes"""
```

Comment:
Please add an explanation of what "resource" is in this context.

Also: Thank you for renaming this. It was VERY confusing.